### ISSUE TITLE
What loss function is used for the NMT example?

### ISSUE THREAD

**Issue by smith-co** [graph4ai/graph4nlp#557]
## ❓ Questions and Help

For the NMT example, what loss function is used? I see Adam optimizer is used. But how the loss is calculated?

Can you please provide some pointers?

**Comment by AlanSwift** [comment]
Cross Entropy Loss
[https://github.com/graph4ai/graph4nlp/blob/1dd4a3cd831f5a991ba122f59ad7dfdd7733f683/graph4nlp/pytorch/models/graph2seq_loss.py#L6](https://github.com/graph4ai/graph4nlp/blob/1dd4a3cd831f5a991ba122f59ad7dfdd7733f683/graph4nlp/pytorch/models/graph2seq_loss.py#L6)

**Comment by code-rex1** [comment]
@AlanSwift is there a way to visualize the loss function? I supposed I have to parse the following input and draw the graph?

```
Epoch 0: [1950 / 11234] loss: 3.798
Epoch 0: [2000 / 11234] loss: 3.852
.....
``

Is this what you would recommend?

**Comment by AlanSwift** [comment]
I suggest you use the tensorboard. It can do it efficiently.
